scripts/read-rust-versions.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Not sure if we need a separate script for this – we could just add its contents to the github workflow directly.